Apps/AccountDataAccess/src/Patient/Api/PatientIdentity.cs

```diff
@@ -0,0 +1,102 @@
+// -------------------------------------------------------------------------
+//  Copyright © 2019 Province of British Columbia
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+// -------------------------------------------------------------------------
+namespace HealthGateway.AccountDataAccess.Patient.Api
+{
+#pragma warning disable SA1600 // Disables documentation for internal class.
+#pragma warning disable SA1602 // Disables documentation for internal class.
+    using System;
+
+    internal record PatientIdentity
+    {
+        public Guid Pid { get; set; }
+
+        public string? Phn { get; set; }
+
+        public string? HdId { get; set; }
```

Comment:
This is for a team discussion regarding consistency of casing:
Usage of **Hdid** (10 definitions)
![image](https://user-images.githubusercontent.com/19548348/235485920-2a2fcde2-1614-4522-80a8-1f8bc126ad74.png)

Usage of **HdId**
![image](https://user-images.githubusercontent.com/19548348/235486000-efe5e84e-1523-472f-9cec-76d0205ebeba.png)

cc: @sslaws

Reply:
HDID is one acronym, not two, so with our casing practices, it should be Hdid.  However, we didn't implement this casing practice until partway through the project, and there are some references we didn't update (especially in the DB models).